skills/declarative/calendarskill/extensions/Microsoft.Bot.Solutions.Extensions/Actions/RetrievePhoto.cs

```diff
@@ -31,26 +31,53 @@ public RetrievePhoto([CallerFilePath] string callerPath = "", [CallerLineNumber]
         [JsonProperty("token")]
         public StringExpression Token { get; set; }
 
+        [JsonProperty("email")]
+        public StringExpression Email { get; set; }
+
+        [JsonProperty("isMe")]
```

Comment:
why we need this in the action?

Reply:
the default photo is different in the two mode. when "isMe" is true, the default photo is a picture with "me". for the other users it will return a photo with their name.